File: SEM 5/MACHINE LEARNING/Practical 01/Q4.R
# Satyajeet Sudhir Dharmadhikari
# TY CSE Roll No. 30
n <- 10
n1 <- 0
n2 <- 1
iteration <- 0

while (iteration < n){
	print(n1)
	n3 <- n1 + n2
	n1 = n2
	n2 = n3
	iteration = iteration +1
}

File: SEM 5/MACHINE LEARNING/Practical 01/Q4.py
#       Dharmadhikari Satyajeet Sudhir
#       TY CSE Roll No. 30

n = 10
n1, n2 = 0, 1
iteration = 0

print("First 10 Fibonacci numbers:")
while iteration < n:
   print(n1)
   n3 = n1 + n2
   n1 = n2
   n2 = n3
   iteration += 1
